### DISCUSSION TITLE
[Feature] Bolt 12 Support

### DISCUSSION THREAD

**Discussion by kn0wmad** [btcpayserver/btcpayserver#4424] [General]
For those CLN folk

Appreciate all your hard work - cheers from all of us at Start9 :)

**Comment by kn0wmad** [comment]
Is Issues the wrong place for feature requests?

**Reply by pavlenex** [reply]
Yes.
![Screenshot 2022-12-10 at 10 52 57](https://user-images.githubusercontent.com/36959754/206847543-0e7320d2-a5ba-418d-b119-dd4199e25f8e.png)

**Comment by kn0wmad** [comment]
In that case, looks like several others need moved as well: 
![image](https://user-images.githubusercontent.com/39687477/207091298-7e124c32-5d55-4f47-afeb-5b4fa1bb3ca4.png)

Would love to hear opinions or plans for BOLT 12, cheers

**Reply by pavlenex** [reply]
These are features verified amount contributors, and have consensus, once the idea is on a roadmap we file an issue.
